Dockerfile

```diff
@@ -202,7 +227,7 @@ COPY . .
 COPY --from=builder /build/sources ./sources/
 COPY --from=grpc /build/grpc ./grpc/
 
-RUN make prepare-sources && cd /build/grpc/cmake/build && make install && rm -rf grpc
+RUN cd /build/grpc/cmake/build && make install && rm -rf /build/grpc
```

Comment:
wouldn't removing prepare-sources here breaks `REBUILD=true`?

Reply:
Ahh, good point, I'll get that back in. I removed it because I was trying to avoid having to put the protobuf compilation tooling in the final container to keep it smaller... I forgot about the rebuild functionality

Reply:
So technically it wouldn't break the `REBUILD=true` functionality, as long as the container had access to the internet to download the sources. But the fact that the protobuf compilers were not in the runtime container did break it.

I have another commit on the way to bring back `prepare-sources` so that non-internet connected containers can still rebuild, and moved the protobuf compiler install to the requirements-core stage so that they will be present in the runtime container as well.